src/ProtectedProxies/MinimalProtectedProxyFactory.sol

```diff
@@ -10,13 +10,14 @@ import {SpherexProtetedMinimalProxy} from "./ProtectedMinimalProxy.sol";
  * @title A factory contract that deploys the SpherexProtetedMinimalProxy contracts.
  */
 contract MinimalProtectedProxyFactory is SphereXConfiguration {
-
     address private immutable IMPLEMENTATION;
     bytes4[] private /*immutable*/ PROTECTED_SIGS; // this is in effect immutable.
-    
-    address private  _allowedDeployer;
```

Comment:
Weird, Why Do I see this line as removed?

Reply:
i have no idea why. look at the code, its there